scripts/_cli.py

```diff
@@ -0,0 +1,207 @@
+
+    df_file = pd.DataFrame()
+
+    for bundlename in bundles_to_download:
+        df = (
+            pd.DataFrame.from_dict(config["databundles"][bundlename]["output"])
+            .rename(columns={0: "filepath"})
+            # .assign(category=config["databundles"][bundlename]["category"])
+            .assign(databundle=bundlename)
+            .set_index("filepath")
+        )
+
+        # Filter out rows where the 'file path' contains '*'
+        df = df[~df.index.str.contains(r"\*", na=False)]
+        df = df[~df.index.str.endswith("/")]
+
+        df_file = pd.concat([df_file, df])
+
+    df_file["retrieve"] = [os.path.isfile(f"{i}") for i in df_file.index]
+
+    console_table(
+        df_file, table_kw={"title": "Databundle Checklist"}
+    )  # [["databundle","retrieve"]]
+
+    missing_bundles = list(df_file.loc[~df_file.retrieve, "databundle"].unique())
+
+    if len(missing_bundles) == 0:
+        console_markdown("No data is missing")
+
+    else:
+        extractable_bundle = [
+            bundlename
+            for bundlename in missing_bundles
+            if set(config["databundles"][bundlename]["urls"].keys()).intersection(
+                ["zenodo", "gdrive", "direct"]
+            )
+        ]
+
+        console_markdown(
+            f"""It looks like there are missing files from: **{", ".join(missing_bundles)}**"""
+        )
+
+        if extractable_bundle:
+            df_link = pd.DataFrame()
+
+            for bundlename in extractable_bundle:
+                df = (
+                    pd.DataFrame(config["databundles"][bundlename]["urls"].items())
+                    .rename(columns={0: "host", 1: "urls"})
+                    .assign(databundle=bundlename)
+                )
+
+                df.loc[df.index[1:], "databundle"] = ""
+                df = df.set_index("databundle")
+
+                df_link = pd.concat([df_link, df])
+
+            console_table(df_link, table_kw={"title": "Retrievable Databundle"})
+
+            console_markdown(
+                f"""The databundle **{", ".join(extractable_bundle)}** can be retrieved either automatically or manually downloaded."""
+            )
+
+        unextractable_bundle = [
+            bundlename
+            for bundlename in missing_bundles
+            if bundlename not in extractable_bundle
+        ]
+
+        if unextractable_bundle:
+            console_markdown(
+                f"""The databundle **{", ".join(unextractable_bundle)}** can only be retrieved automatically."""
+            )
+
+    console_markdown(
+        f"""
+    Options:
+
+    - **check**: update the Databundle Checklist to see if the file is included
+    - **all**: retrieve all missing databundles, namely **{", ".join(missing_bundles)}**
+    - **rerun**: retrieve all databundles again, namely **{", ".join(bundles_to_download)}**
+    - **bundle_...**: retrieve the selected databundles, can be more than one
+    - Press **ENTER** to end this loop
+    """
```

Comment:
A love the idea of having such a report! May it be a good idea to try make the message more human-readable? The intended audience of this script are people who are starting to use the model and are not aware of our jargon 🙂 Though, that is just a detail which will be probably more relevant when polishing the implementation